service/service.go

```diff
@@ -271,7 +271,7 @@ func New(config Config) (*Service, error) {
 	var frameworkBackOff *backoff.ExponentialBackOff
 	{
 		frameworkBackOff = backoff.NewExponentialBackOff()
-		frameworkBackOff.MaxElapsedTime = 5 * time.Minute
+		frameworkBackOff.MaxElapsedTime = 0 // retry forever
```

Comment:
It looks like `MaxElapsedTime` is quite useless. Based on the godoc I understand max elapsed time is considered during creation of the backoff and not since the first error. I would have also assumed the backoff to be reset after it was working again but this also does not seem to be the case. So disabling and retrying forever based on the exponential delay seems to be the way to go now. 

> If the time elapsed since an ExponentialBackOff instance is created goes past the MaxElapsedTime, then the method NextBackOff() starts returning backoff.Stop.

See https://godoc.org/github.com/cenkalti/backoff#ExponentialBackOff

Reply:
> Based on the godoc I understand max elapsed time is considered during creation of the backoff and not since the first error.

That what I was expecting. What was the problem? Operations taking longer than 5min?

Reply:
The backoff is created/initialized during program start. Once the operator is running for five minutes and an error occurs it will not retry anymore.

Reply:
Ack. Good you found that.

I think it'd be nice `WithMaxTries`, so the operator fails to start at some point.

Reply:
Lets continue the discussion here: https://github.com/giantswarm/cert-operator/pull/105#pullrequestreview-76063068.